cmd/build.go

```diff
@@ -157,7 +171,53 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			if entry.UUID == "" {
+				fmt.Println("sd-local collects UUIDs for statistical surveys.")
+				fmt.Println("You can reset it later by removing the UA key from config.")
+				fmt.Print("Would you please cooperate with the survey? [y/N]: ")
+
+				input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+				if err != nil {
+					return err
+				}
+				input = strings.TrimSuffix(input, "\n")
+				if input == "y" || input == "Y" || input == "yes" || input == "Yes" {
+					uuidObj, err := uuid.NewUUID()
+					if err != nil {
+						return err
+					}
+					uuidStr = uuidObj.String()
+
+					err = entry.Set("uuid", uuidStr)
+					if err != nil {
+						return err
+					}
+
+					err = config.Save()
+					if err != nil {
+						return err
+					}
+					fmt.Printf("UUID key has been added to %s\n", configPath)
+				} else {
+					err = entry.Set("uuid", "-")
```

Comment:
Here(https://github.com/screwdriver-cd/sd-local/pull/57/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R184-R186), if the value is empty, it seems to be automatically set to "-", so I think it is okay to pass an empty character as the argument.

Reply:
I want to use it to check if he or she has answered y/n.
https://github.com/screwdriver-cd/sd-local/pull/57/files#diff-5794c40aa4c9301aaf9c39eacf7d3c8511f2511cb24bdd95e0a08704ba583a5fR174

Reply:
Ah, I misunderstand. But I think I don't need to do that. It's seems that a misleading notation.

Reply:
I see.